rust/ql/lib/codeql/rust/internal/PathResolution.qll

```diff
@@ -648,6 +648,19 @@ final class ImplItemNode extends ImplOrTraitItemNode instanceof Impl {
 
   override Visibility getVisibility() { result = Impl.super.getVisibility() }
 
+  TypeParamItemNode getBlanketImplementationTypeParam() {
+    result = this.resolveSelfTy() and
+    result = super.getGenericParamList().getAGenericParam() and
```

Comment:
Is this check actually needed?

Reply:
It makes sense, but I've checked a few DBs and it doesn't make a difference. Should we take it out?